utils/utils.ts

```diff
@@ -27,3 +27,30 @@ export function parseQueryAmount(query: ParsedUrlQuery) {
     currency: currency?.toUpperCase() || "USD",
   }
 }
+
+export function parseDisplayCurrency(query: ParsedUrlQuery) {
+  const display = query.display as string | null
+
+  return {
+    display: display ?? localStorage.getItem("display") ?? "USD",
+  }
+}
+
+export function safeAmount(amount: any, hasLeadingZeros?: boolean, unit?: any) {
```

Comment:
Why is `amount` and `unit` type any since we know what types they return?

Reply:
cleaned up typings in latest commit